packages/libs/eda/src/lib/core/components/variableTrees/VariableList.tsx

```diff
@@ -958,8 +958,9 @@ const FieldNode = ({
         isMultiPick
           ? ''
           : isDisabled
-          ? customDisabledVariableMessage ??
-            'This variable cannot be used with this plot and other variable selections.'
+          ? (customDisabledVariableMessage ??
```

Comment:
**question**: I might be doing something wrong, but I think the string will interpolate the boolean from the lefthand side of the `??`

I.e., 

<img width="659" alt="image" src="https://github.com/VEuPathDB/web-monorepo/assets/24446573/4750faf3-e967-490b-9a39-614e5b8ef565">

Reply:
Why, hello, stranger!

The type of `customDisabledVariableMessage` is `customDisabledVariableMessage: string | undefined`, so it shouldn't ever be `true` or `false`.

I was hoping that typescript would not compile if we changed that to `customDisabledVariableMessage: string | boolean | undefined`, but [it doesn't appear to](https://www.typescriptlang.org/play?#code/CYUwxgNghgTiAEYD2A7AzgF0QV00gtgCICWaUARhCMAGqzEVUCyIaZA5iAFzyYzEp28AD7xySJFSgoR8bClAAzAdQDcAWABQW5OiwAPeAF54ACjC4MBEmUrU6-RiBZsoneAH4PW+L7---AHIAFQALUngAN3onRGkUJCxyBFxqeAB3YgxQ+GyIgAcIRPhpYHhE0JAYKJi7XhAqMAxiVDQAOkCASngAah8AgcD4JmxmgFpoiGw06Mc7NBK4eASsCxg4FAwIAE9ebHz8pBgMag7VIA).

Thus, I will change this to a ternary expression. Thanks for the review, @adnauseum!

Reply:
It's worth mentioning that, even with the ternary expression, if `customDisabledVariableMessage` is `true`, then the resulting message will be

```
true Muti-valued variables are not currently supported.
```

Reply:
Good call! 🫡 

And hello 🙂. Hope all has been well, especially with the summer kids!